apm-opentracing/src/main/java/co/elastic/apm/opentracing/ElasticApmTracer.java

```diff
@@ -77,4 +84,9 @@ public SpanBuilder buildSpan(String operationName) {
         }
         return null;
     }
+
+    @Override
+    public void close() {
+        scopeManager().active().close();
```

Comment:
call `co.elastic.apm.agent.impl.ElasticApmTracer#stop` here

Reply:
How to call the ElasticApmTracer#stop?, cannot call package from submodule apm-opentracing :confused: :question:

Reply:
What I mean by that is that you should create another instrumentation and call the tracer from there.

Reply:
Ok